tests/shared/src/test/scala/scala/meta/tests/parsers/TopLevelTermSuite.scala

```diff
@@ -0,0 +1,82 @@
+package scala.meta.tests.parsers
+
+import scala.compat.Platform.EOL
+import scala.meta._
+
+import munit._
+
+class TopLevelTermSuite extends FunSuite {
+  test("allowToplevelTerms simple") {
+    implicit val dialect: Dialect = dialects.Scala3.withAllowToplevelTerms(true)
+
+    val sourceString = """def foo(x: Int): Int = x
+                          |foo(x)
+""".trim.stripMargin
```

Comment:
this is a weird string constant. indent all lines, including last, use margin char `|`

Reply:
Done.